components/mailgun/mailgun.app.js

```diff
@@ -0,0 +1,192 @@
+const axios = require("axios");
+const get = require("lodash/get");
+const moment = require("moment");
+const retry = require("async-retry");
+
+module.exports = {
+  type: "app",
+  app: "mailgun",
+  propDefinitions: {
+    domain: {
+      type: "string",
+      label: "domain",
+      description: "Domain used to send and receive email.",
+      async options(context) {
+        const q = context.query;
+        const options = [];
+        const results = await this.getAllDomains();
+        for (const domains of results) {
+          options.push(domains.name);
+        }
+        return options;
+      },
```

Comment:
Is q used here? I think you can shorten this:
```
async options() {
  const domains = await this.getAllDomains();
  return domains.map((domain) => domain.name);
}
```

Reply:
No, `q` is not used. You are right. This was shorten.